ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.14.yaml

```diff
@@ -641,6 +641,14 @@ tests:
     - ref: ipi-install-heterogeneous
     - ref: openshift-e2e-test
     workflow: openshift-e2e-gcp-ovn
+- as: ocp-e2e-sno-power
+  cron: 0 4 * * *
+  steps:
+    env:
+      LIVE_INITRAMFS_URL: https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/pre-release/4.14.0-rc.0/rhcos-live-initramfs.ppc64le.img
```

Comment:
Is this something we'll need to link to a sym-link to make sure it stays up to date?

Reply:
I believe it's not required since the latest dir on each release will contain latest images only. I have modified the urls to point to the latest.
https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.14/